docs/en/security-privacy-considerations.rst

```diff
@@ -0,0 +1,495 @@
+
+The Issuer's information is displayed to the User during the issuance process and can be subsequently read by the User as it is inside the issued Credential. 
+In addition to the Issuer's information, the Digital Credential Metadata Type¶ also contains information on the Authentic Source.
+
+SR-W-30
+~~~~~~~
+.. list-table:: 
+   :widths: 8 92
+
+   * - |check-icon|
+     - The Wallet must provide trustworthy and complete information about Verifiers to the User.
+
+The Wallet validates the OpenID Federation Trust Chain related to the Verifier and its information is displayed to the User before the presentation.
+
+Privacy Requirements
+--------------------
+
+PR-CF-30
+~~~~~~~~
+.. list-table:: 
+   :widths: 8 92
+
+   * - |check-icon|
+     - The Credential Format must ensure that there is a robust mechanism to ensure that data that is not to be released to a Verifier cannot be extracted by the Verifier (selective disclosure).
+
+Both SD-JWT and mDOC-CBOR provide selective disclosure capability, allowing Holders to reveal only specific fields to the Verifier. 
+
+PR-CF-40
+~~~~~~~~
+.. list-table:: 
+   :widths: 8 92
+
+   * - |partially-check-icon|
+     - The Credential Format must support correlation protection.
+
+While selective disclosure is a strong tool for preventing correlation, full unlinkability is not guaranteed in all cases. Issues like Verifier collusion or Issuer tracking can arise, especially if sensitive identifiers (e.g., taxpayer ID) are disclosed. 
+
+.. tip::
+  Batch issuance, using different key binding keys and salts for each Credential, can mitigate Verifier/Verifier and presentation unlinkability risks. 
+
+PR-E-60
+~~~~~~~
+.. list-table:: 
+   :widths: 8 92
+
+   * - |check-icon| 
+     - The Trust Framework must ensure that the Issuer cannot learn where the User uses the Credential.
+
+When a Verifier performs the Trust Evaluation of the Issuer of a Credential following [`OID-FED`_], the Issuer cannot know who is the User presenting the Credential.
+In addition, privacy is protected also during the check of the Credential's status. By using Status Assertion [`OAUTH-STATUS-ASSERTION`_], the IT Wallet specification ensures 
+that while the Verifier checks the Credential's validity, the Issuer does not learn where or when the Credential is being used.
+
+PR-E-70
+~~~~~~~
+.. list-table:: 
+   :widths: 8 92
+
+   * - |partially-check-icon|
+     - The Trust Framework must support correlation protection.
+
+In the case of IT Wallet, as the Trust Framework is based on OpenID Federation [`OID-FED`_], the IT Wallet specification has the following mechanisms in place that could help to reduce the correlation:
+
+- *Verifier-Verifier*: OpenID federation provides a way to check whether the Verifier is asking from the Wallet the information that is authorized to ask or not.  Thus, it will minimize the data exchange and consequently avoid User profiling by colluding between two Verifiers;
```

Comment:
```suggestion
- *Verifier-Verifier*: OpenID Federation provides the evaluation mechanisms to verify whether the Verifier is asking from the Wallet the information that is authorized to ask or not.  Thus, it will minimize the data exchange and consequently avoid User profiling by colluding between two Verifiers;
```

considering the strong push back got in federation wallet arch specs, we should evaluate to abandon the presentation_definitions_supported and use some more general approach based on trust marks (alos more close to other policies format be included in x.509 extensions).